src/app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -65,9 +65,14 @@ protected function validator(array $data)
     protected function create(array $data)
     {
         return User::create([
```

Comment:
Your should return a `UserResource`

It should look something like this 
```php
return new UserResource(User::create([..]));
```
Resources aren't yet in the project. Once #94 is merged, you can rebase your branch so you can use them

Reply:
Fixed in 645b86445a62a2f926c36d73fefab764ee1b7ffd